pkg/tuned/controller.go

```diff
@@ -42,9 +42,20 @@ const (
 	scWarn
 	scError
 	scSysctlOverride
+	scReloading // reloading is true during the TuneD daemon reload.
 	scUnknown
 )
 
+const (
+	// Constants used for controlling TuneD;
+	// they will be set to 2^0, 2^1, 2^2, ..., 2^n
```

Comment:
this general comment is probably best fit above `const` on line 49 (applies to other consts e.g. like lines 38-47)